spec/lib/dyph3/differ_spec.rb

```diff
@@ -10,9 +10,9 @@
           left =  "ants bears cat dog".split
           base =  "ants bears cat dog".split
           right =  "ants elephant cat bears dog".split
-          result = Dyph3::Differ.merge_text(left, base, right, split_function: identity , join_function: identity, current_differ: current_differ)
-          expect(result[0]).to eq right
-          expect(result[1]).to be false
+          merge_result = Dyph3::Differ.merge_text(left, base, right, split_function: identity , join_function: identity, current_differ: current_differ)
+          expect(merge_result.joined_results).to eq right
+          expect(merge_result.success?).to be true
```

Comment:
@jelder  When I get time to refactor the original_heckell_differ class I will refactor the specs a bit.